src/garage/np/algos/cem.py

```diff
@@ -66,18 +79,14 @@ def __init__(self,
         self.all_returns = []
         self.all_params = [self.cur_mean.copy()]
         # fixed
```

Comment:
Also this comment is not saying anything I believe? Please remove or add information.

Reply:
It actually means 'constant', in comparison to previous comments '(updated) epoch-wise' and '(updated) epoch-cycle-wise'. Please help me fix the bad wording @zequnyu